spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
You mean using `ApplicationMailer.prepend_view_path` in this spec? Will that be not thread safe?